README.md

```diff
@@ -352,3 +352,4 @@ Team in the Inaugural DIHARD Challenge](https://www.isca-speech.org/archive/pdfs
 | IBM     | [Watson Speech To Text API](https://www.ibm.com/watson/services/speech-to-text) |
 | DeepAffects | [Speaker Diarization API](https://www.deepaffects.com/diarization-api) |
 | Alibaba | [Tingwu (听悟)](https://tingwu.aliyuncs.com/trans) |
+| Microsoft | [Speaker Diarization API](https://azure.microsoft.com/en-us/services/cognitive-services/speaker-recognition/) |
```

Comment:
This is not a diarization API, but a speaker recognition API.

Reply:
Hi @wq2012, I'm currently studying this API and in the documentation they mention the diarization feature. Could you check please? https://learn.microsoft.com/en-us/azure/cognitive-services/speech-service/conversation-transcription

Reply:
Yes this doc indeed mentioned diarization. But the link in your original PR is not about diarization at all. Please update it.